packages/app/src/systems/Send/machines/sendMachine.ts

```diff
@@ -134,18 +146,23 @@ export const sendMachine = createMachine(
           if (!to || !assetId || !amount || !network?.url || !account) {
             throw new Error('Missing params for transaction request');
           }
-          const wallet = new WalletLockedCustom(account.address, network.url);
-          const createOpts = { to, amount, assetId };
-          const transactionRequest = await TxService.fundTransaction({
-            transactionRequest: await TxService.createTransfer(createOpts),
-            wallet,
-          });
+          try {
+            const wallet = new WalletLockedCustom(account.address, network.url);
+            const createOpts = { to, amount, assetId };
+            const transactionRequest = await TxService.fundTransaction({
+              transactionRequest: await TxService.createTransfer(createOpts),
+              wallet,
+            });
 
-          return {
-            address: wallet.address.toString(),
-            transactionRequest,
-            providerUrl: network.url,
-          };
+            return {
+              address: wallet.address.toString(),
+              transactionRequest,
+              providerUrl: network.url,
+            };
+          } catch (e) {
+            await ReportErrorService.handleError(e);
+            throw new Error('Failed to create transaction request');
+          }
```

Comment:
same thing here, `TxService.fundTransaction` should handle it's own error conditions.

then reporting should be handled in `fetchMachine`